.github/workflows/version-bump.yml

```
@@ -116,29 +116,33 @@ jobs:
-            git rebase --abort
-            echo "Manual intervention may be required"
-            exit 1
-          fi
-
-          # Push using force-with-lease for safety (only if remote hasn't changed since fetch)
-          echo "⬆️ Pushing rebased commit with force-with-lease..."
-          git push origin HEAD:main --force-with-lease
-
-          echo "✅ Successfully pushed version ${{ steps.version-info.outputs.new_version }}"
+          echo "📤 Creating version bump branch and PR..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)